flang/test/Lower/OpenMP/parallel-reduction-array.f90

```diff
@@ -0,0 +1,74 @@
+! RUN: bbc -emit-hlfir -fopenmp -o - %s 2>&1 | FileCheck %s
+! RUN: %flang_fc1 -emit-hlfir -fopenmp -o - %s 2>&1 | FileCheck %s
+
+program reduce
+integer, dimension(3) :: i = 0
+
+!$omp parallel reduction(+:i)
+i(1) = 1
+i(2) = 2
+i(3) = 3
```

Comment:
Is the test written to deliberately not use the reduction operation in the body?

Reply:
I was just trying to create a test with a very simple body. Do you want a test with the reduction operation in the body as well?

Reply:
Yes, it would be good to have the test match the default way that it will be used.